README.md

```diff
@@ -43,36 +68,66 @@ An alternative is to install [Azure CLI](https://docs.microsoft.com/en-us/cli/az
    echo "---" >> cmd/clusterctl/examples/azure/out/provider-components.yaml
    kustomize build vendor/sigs.k8s.io/cluster-api/config/default/ >> cmd/clusterctl/examples/azure/out/provider-components.yaml
    ```
-3. Create the cluster. 
-   Kubernetes Version >= 1.11 is required to enable CRD subresources without needing a feature gate.
+3. Create the cluster.
 
+   **NOTE:** Kubernetes Version >= 1.11 is required to enable CRD subresources without needing a feature gate.
 
     **Linux**
     ```bash
-    ./bin/clusterctl create cluster --provider azure \
-    -m cmd/clusterctl/examples/azure/out/machines.yaml \
-    -c cmd/clusterctl/examples/azure/out/cluster.yaml \
-    -p cmd/clusterctl/examples/azure/out/provider-components.yaml \
-    --vm-driver kvm2 --minikube kubernetes-version=v1.12.2
+    ./bin/clusterctl create cluster -v 3 \
```

Comment:
Looks like the commands are the same for both mac and linux now. (Thanks to Kind!) . Let's just keep one :)